evictionfree/declaration_sending.py

```diff
@@ -253,6 +241,29 @@ def send_declaration(decl: SubmittedHardshipDeclaration):
     if user.email:
         send_declaration_to_user(decl, pdf_bytes)
 
+    if decl.fully_processed_at is None:
+        user.chain_sms_async(
+            [
+                _(
+                    "%(name)s, you can download a PDF of your completed declaration form by "
+                    "logging back into your account: %(url)s."
+                    % {
+                        "name": user.first_name,
+                        "url": f"https://www.evictionfreeny.org/{user.locale}/login",
```

Comment:
oh www.evictionfreeny.org shouldn't be hard-coded here!  we should be getting it from, um, `site_utils` i think.  that way this will be demo.evictionfreeny.org if they're on demo.

Reply:
on it!